### PR TITLE
Added timeout for repeating Twitch stream promotion

### LINKED ISSUE #11: [bug] Repeated promotion notifications on stream reset
There should probably be a timeout after a stream notification before someone can be promoted again.

This should be in memory, no need to bother the database with this.

### PR DESCRIPTION
**Pull request by notfooddev**

### CODE REVIEW

File: src/commands/config.ts [hunk 1]
```diff
@@ -45,6 +45,8 @@ export default class ConfigCommand implements ICommand {
 				streamShoutString = `${shoutChannel?.name || 'ERR-404'} (${ss.streamShout})`;
 			}
 
+			let streamTimeoutString = ss.streamTimeout + " hours";
```
Comment (WesVleuten) [inline src/commands/config.ts]: Should return `Off` if set to 0. To be more inline with the rest of the settings

File: src/events/presenceUpdate.ts [hunk 2]
```diff
@@ -26,6 +27,21 @@ export default async function PresenceUpdateEvent(discordClient: DiscordClient,
 	}
 
 	const whiteListed = await CheckGameWhitelisted(streamingActivity);
+	
+	if ((serverSettings.streamLiveRole !== null || serverSettings.streamShout !== null) && whiteListed) {
```
Comment (WesVleuten) [inline src/events/presenceUpdate.ts]: Should check if `streamTimeout > 0`, right?

**Review by WesVleuten — CHANGES_REQUESTED** [review]
Looks great, just a couple small tweaks

File: src/events/presenceUpdate.ts (hunk 2 above)
Comment (notfooddev) [reply src/events/presenceUpdate.ts]: Not necessarily, but it would be a good optimization regardless

**Review by WesVleuten — APPROVED** [review]

**Pull request merged**